core_data_modules/cleaners/somali/demographic_cleaner.py

```diff
@@ -97,3 +97,35 @@ def clean_age(cls, text):
             return cleaned_words
 
         return Codes.NOT_CODED
+
+    @staticmethod
+    def clean_age_range(age, min_age_inclusive=10, max_age_inclusive=99):
```

Comment:
s/clean_age_range/clean_age_within_range/

When I first read this, I was expecting it to be something like "20-29"

Reply:
That's a much better name, thanks. Changed.